uint8 id

float32 roll
float32 pitch
float32 yaw
float32 accl_x
float32 accl_y
float32 accl_z
float32 gyro_x
float32 gyro_y
float32 gyro_z
float32 mag_x
float32 mag_y
float32 mag_z
